en/guides/developers/tested-configure-new-programming-language/README.md

```diff
@@ -0,0 +1,1187 @@
+}
+```
+
+### Typeconfiguratiebestand
+Het typeconfiguratiebestand is een JSON-bestand met de benamingen van datatypes en gebruikte conventies voor het
+visualiseren van een codeprompt.
+Dit configuratiebestand wordt gebruikt om sjabloonopgaven te instantiëren voor de programmeertaal.
+Maak eerst het configuratiebestand aan: `tested/languages/c/types.json`.
+
+#### Console informatie
+- `console.name`:
+  De gebruikt naam van de programmeertaal in de console prompt.
+  Bijvoorbeeld: `python` voor Python en `c` voor C.
+- `console.prompt`:
+  Het prompt symbool voor de programmeertaal.
+  Bijvoorbeeld: `>>>` voor Python en `>` voor C.
+
+```json
+"console": {
+  "name": "c",
+  "prompt": ">"
+},
+```
+
+#### Haakjes
+De gebruikte haakjes voor collectie datatypes.
+Bijvoorbeeld: `[]` voor de python collecties, `<>` voor de generieke Java objecten behalve arrays die `[]` gebruiken.
+C in TESTed ondersteund geen collecties, dus zullen Java en Haskell als voorbeeld nemen.
+- `brackets.open`: Generiek openingshaakje
+- `brackets.clase`: Generiek sluitingshaakje
+
+> Voorbeeld Java lijst: `List<?>`
+
+- `brackets.<collection>.open`: Specifiek openingshaakje voor de gegeven collectie
+- `brackets.<collection>.close`: Specifiek sluitingshaakje voor de gegeven collectie
+
+> Voorbeeld Java integer array: `int[]`
+> 
+> Voorbeeld Haskell integer lijst: `[Int]`
+
+#### Datatypes
+Naast de console informatie en haakjes moeten ook de benaming van de ondersteunde datatypes worden opgegeven.
+```json
+"integer": "int",
+"rational": "double",
+"char": "char",
+"text": "char*",
+"boolean": "bool",
+"nothing": "void",
+"int8": "signed char",
+"uint8": "unsigned char",
+"int16": "short",
+"uint16": "unsigned short",
+"int32": "int",
+"uint32": "unsigned int",
+"int64": "long",
+"uint64": "unsigned long",
+"bigint": "long long",
+"single_precision": "float",
+"double_precision": "double",
+"any": "void*",
+"inner": {
+},
+```
+
+Ondersteunde waarden:
+- **string**: Naam van het TESTed datatype in de programmeertaal.
+- **boolean**: Alleen voor collectie datatypes waarbij de notatie van het collectietype uitsluitend met haakjes gebeurt.
+  - `true`: Het datatype van de gegevens in de collectie bevindt zich tussen de haakjes.
+    Hierbij moeten de haakjes van het collectie type expliciet worden opgeven.
+    Bijvoorbeeld Haskell lijst: `[Int]`
+  - `false`: De haakjes bevinden zich achter het datatype van de gegevens in de collectie.
+    Hierbij moeten de haakjes van het collectie type expliciet worden opgeven.
+    Bijvoorbeeld Java array: `int[]`
+
+`inner` is het object waarbij elk datatype, die een andere naam moet krijgen in een collectie, moet opgegeven worden.
+Dit geldt niet voor datatypes waarbij de haakjes achter het datatype geplaatst worden.
+Bijvoorbeeld Java: `Integer` in plaats van `int` voor `int32`.
+
+#### Natuurlijke taal constructies
+De benaming in de natuurlijke taal voor de collectie datatypes en tekst, is afhankelijk van de programmeertaal.
```

Comment:
```suggestion
De benaming in de natuurlijke taal voor de collectiedatatypes en tekst, is afhankelijk van de programmeertaal.
```

What is the "tekst" part here? As far as I can see, we only specify data type names in the section of the guide.